responsibleai/model_monitor/components/src/calculate_attribution.py

```diff
@@ -0,0 +1,177 @@
+    else: 
+        lgbm = LGBMRegressor(boosting_type='gbdt', learning_rate=0.1,
+                            max_depth=5, n_estimators=200, n_jobs=1, random_state=777)
+    model = lgbm.fit(X, y)
+    
+    _logger.info("Created lightgbm model")
+    return model
+
+
+def get_model_wrapper(task_type, target_column, baseline_df, production_df):
+    """Create model wrapper using ml-wrappers on which to calculate feature importances
+
+    Args:
+      task_type: str, The task type (regression or classification) of the resulting model
+      target_column: str, the column to predict
+      baseline_df: The baseline dataframe
+      production_df: The production dataframe
+
+    Returns:
+       An appropriate model wrapper
+    """
+    y_train = baseline_df[target_column]
+    x_train = baseline_df.drop([target_column], axis=1)
+    x_test = production_df.drop([target_column], axis=1)
+    model = create_lightgbm_model(x_train, y_train, task_type)
+
+    if task_type == 'classification':
+        all_data = pd.concat([x_test, x_train])
+        model_predict = model.predict(all_data)
+        model_predict_proba = model.predict_proba(all_data)
+        model_wrapper = PredictionsModelWrapperClassification(
+            all_data,
+            model_predict,
+            model_predict_proba)
+    else:
+        all_data = pd.concat([x_test, x_train])
+        model_predict = model.predict(all_data)
+        model_wrapper = PredictionsModelWrapperRegression(all_data, model_predict)
+
+    _logger.info("Created ml wrapper")
+    return model_wrapper
+
+
+def compute_categorical_features(baseline_df, target_column):
+    """Compute which features are categorical based on data type of the columns.
+
+    Args:
+      baseline_df: The baseline dataframe
+      target_column: str, the column to predict
+
+    Returns:
+      list: A list of categorical features.
+    """
+    categorical_features = []
+    for column in baseline_df.columns:
+        baseline_column = pd.Series(baseline_df[column])
+        if baseline_column.dtype.name == 'object' and baseline_column.name != target_column:
+            categorical_features.append(baseline_column.name)
+    _logger.info("Successfully categorized columns")
+    return categorical_features
+
+
+def compute_explanations(model_wrapper, df, categorical_features, target_column, task_type):
+    """Compute explanations (feature importances) for a given dataset
+
+    Args:
+      model_wrapper: wrapper around a model that can be used to calculate explanations
+      df: The dataframe
+      categorical_features: list of categorical features not including the target column
+      target_column: str, the column to predict
+      task_type: str, the task type (regression or classification) of the resulting model
+
+    Returns:
+      list: A list of explanations.
+    """
+    # Create the RAI Insights object, use either baseline as train and test data
```

Comment:
```suggestion
    # Create the RAI Insights object, use baseline as train and test data
```
This seems like it doesn't belong there.